app/src/main/java/org/torproject/android/ui/onboarding/CustomBridgesActivity.java

```diff
@@ -86,10 +87,8 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
         findViewById(R.id.btShareQr).setOnClickListener(v -> {
             String setBridges = Prefs.getBridgesList();
             if (!TextUtils.isEmpty(setBridges)) {
-                try {
-                    integrator.shareText("bridge://" + URLEncoder.encode(setBridges, "UTF-8"));
-                } catch (UnsupportedEncodingException e) {
-                    e.printStackTrace();
+                if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
```

Comment:
Again why the limit for Android 13 on this call?

Reply:
Passing a different UTF parameter to the ```
URLEncoder.encode()
``` caused required SDK Level 33 so I now reverted to the old param.